package-res/resources/web/pentaho/lang/EventSource.js

```diff
@@ -324,11 +325,12 @@ define([
       try {
         result = this._emit(event);
       } catch(e) {
-        if (event) {
-          console.log("Exception thrown during '", (event.type || "NO_TYPE"), "' loop:", e);
+        if(event) {
+          logger.log("Exception thrown during '" + (event.type || "NO_TYPE") + "' loop");
         } else {
-          console.log("Exception thrown due to an invalid event:", e);
+          logger.log("Exception thrown due to an invalid event");
         }
+        logger.log(e);
```

Comment:
I'd say this is a warning level message. Not an error, cause we're not aborting the show. But definitely not log.